screen/settings/IsItMyAddress.tsx

```
@@ -0,0 +1,283 @@
+      });
+    }
+  }, [matchingWallets]);
+
+  const renderFormattedText = (text: string, values: { [key: string]: string }) => {
+    const regex = /\{(\w+)\}/g;
+    const parts = [];
+    let lastIndex = 0;
+    let match;
+    let index = 0;
+
+    while ((match = regex.exec(text)) !== null) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)